packages/core/src/config.js

```diff
@@ -190,6 +190,10 @@ export const configSchema = {
         type: 'boolean',
         default: false
       },
+      captureAllSrcsetUrls: {
```

Comment:
```suggestion
      captureSrcset: {
```
easier to type and conveys same [ as we dont plan to introduce capture a few from src set ]